src/test/java/com/vaadin/testbenchexample/junit5/WrongBrowserIT.java

```diff
@@ -0,0 +1,30 @@
+package com.vaadin.testbenchexample.junit5;
+
+import org.junit.jupiter.api.Assumptions;
+
+import com.vaadin.testbench.BrowserTest;
+import com.vaadin.testbench.annotations.RunLocally;
+import com.vaadin.testbench.parallel.Browser;
+import com.vaadin.testbench.parallel.BrowserUtil;
+
+/**
+ * In this example you can see how to filter test to run only on selected
+ * browser. {@code Capabilities} are being injected by TestBench.
+ */
+@RunLocally(Browser.CHROME) // the driver (could be multi-browser remote driver - Sauce)
+public class WrongBrowserIT extends AbstractJUnit5IT {
+
+    @BrowserTest // that test will be ignored
+    public void clickButton_notificationShown_safari() {
+        Assumptions.assumeTrue(BrowserUtil.isSafari(getCapabilities()),
```

Comment:
Can we also show an example of injected `Capabilities` as a method's argument?

Reply:
Sure